sample/app.yaml

```diff
@@ -0,0 +1,1712 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Modified by Google LLC
+
+# Portions of this file are based on code from the OpenTelemetry Demo:
+# https://github.com/GoogleCloudPlatform/opentelemetry-demo
+---
+apiVersion: v1
```

Comment:
add the `opentelemetry-demo` namespace manifest to this file too (I think the upstream demo already has that)